apps/portal/src/app/Header.tsx

```diff
@@ -98,9 +98,13 @@ export const connectLinks = [
 
 const supportLinks = [
   {
-    name: "Support Site",
+    name: "Get thirdweb support",
     href: "https://thirdweb.com/support",
   },
+  {
+    name: "Knowledge Base",
+    href: "https://knowledgebase.thirdweb.com",
```

Comment:
getting NXdomain when trying to hit this, is it set up?

Reply:
I was going to move our support page from support.thirdweb.com to knowledgebase.thirdweb.com but realize it will break a lot of links of thirdweb.com/support, hurt existing SEO, and may not actually make a big difference. I reverted back to support.thirdweb.com.